pkg/reconcilerevent/cluster.go

```diff
@@ -0,0 +1,36 @@
+package reconcilerevent
```

Comment:
any particular reason this isn't an internal package?

Reply:
At first I was thinking that this might be more of an 'external' API. But it really isn't as we're never going to publish it externally. I'll move this back to `internal`.